cradle-core/src/main/java/com/exactpro/cradle/CradleManager.java

```diff
@@ -56,7 +63,7 @@ public void init(String instanceName, boolean prepareStorage) throws CradleStora
 	
 	public void init(String instanceName) throws CradleStorageException
 	{
-		initStart(instanceName, false);
+		initStart(instanceName, false, 0, 0);

```

Comment:
Maybe use default values instead of `0, 0`

Reply:
Agreed, done.